Update support for engines variable in package.json for node version detection. (#114)
codesandbox/docs: packages/projects-docs/pages/learn/environment/vm.md

**nodeVersion**

```json
/* .codesandbox/environment.json */

{
  "nodeVersion": "14"
}
```

Single value configuration, type `string`, by default it is set to `"16"` .

<Callout emoji="*">

> If your package.json uses [engines](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) field then you need any `.nvmrc` or `environment.json` files. 

Reviewer: ```suggestion
If your package.json uses [engines](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) field then a `.nvmrc` or `environment.json` file is not needed. 
```
